### DISCUSSION TITLE
How to grab an array value

### DISCUSSION THREAD

**Discussion by sureMOISE** [mockoon/mockoon#879] [Help]
is it possibile to get the value from a variable array?
example:
```javascript
{{setVar 'refLat' (faker 'address.latitude' 46.722873 46.713458 8)}}
{{setVar 'refLong' (faker 'address.longitude' 8.121358 8.102806 8)}}
{{setVar 'coords' (faker 'address.nearbyGPSCoordinate' (array (parseInt @refLat) ( parseInt @refLong)) 1 true)}}
```
the variable `@coords` return `['46.722873', '8.102806']` and I need to retrive the first and the second value in an object, in something like:

```json
{
  "latitude": {{{@coords[0]}}},
  "longitude":  {{{@coords[1]}}}
}
```
the final result should be:
```json
{
  "latitude": 46.722873,
  "longitude":  8.102806
}
```

 any idea? thanks

**Comment by 255kb** [comment]
It should work by doing `{{@coords.[0]}}`. Unfortunately, the `faker` helper stringify everything, which shouldn't be the case. So, you don't store an array in `coords` but a string. 
I will open a new issue and investigate what is the best fix for this. 
In the meantime you could do some split and substring, but it's not ideal.

**Reply by sureMOISE** [reply]
Thanks @255kb, while I wait a possibile fix, I will try some other options...
